### DISCUSSION TITLE
Add OpenTelemetry to comparison doc

### DISCUSSION THREAD

**Discussion by blkperl** [prometheus/prometheus#13928] [Q&A] [answered]
### Proposal

Hello! I think it would be helpful if OpenTelemetry was discussed in the comparison doc. Thanks!

https://prometheus.io/docs/introduction/comparison/

**Comment by machine424** [comment]
The page in question compares Prometheus with other time series databases. OpenTelemetry isn’t a time series database, nor does it provide/offer one. However, Prometheus and OpenTelemetry can be used in tandem. For more information, you can check https://prometheus.io/blog/2024/03/14/commitment-to-opentelemetry/.

**Comment by blkperl** [comment]
Hi @machine424, Thanks for responding. I think I'm looking for a comparison between the collectors specifically. 

- https://opentelemetry.io/docs/collector/
- https://opentelemetry.io/docs/kubernetes/operator/target-allocator/

My understanding is that one of the tradeoffs of using OpenTelemetry for collection is that we lose the persistence function of Prometheus's tsdb.

The use case is we need metrics to go to a third party service but we also need metrics locally for a short duration for things like Kubernetes Horizontal Pod Scaling with custom metrics. We are running a prometheus setup with 25 cores and 220Gi of memory and our calculations suggest that the OTEL collector would only need half the resource usage across multiple replicas. Unfortunately, we would still likely need to maintain a prometheus cluster as well to maintain the metrics needed for scaling.

**Comment by machine424** [comment, marked as answer]
> My understanding is that one of the tradeoffs of using OpenTelemetry for collection is that we lose the persistence function of Prometheus's tsdb.

Otel can export data into various backends, including Prometheus, for persistence https://opentelemetry.io/docs/concepts/components/#exporters. I suggest you explore other sections of the OpenTelemetry documentation as they are quite informative.

> The use case is we need metrics to go to a third party service but we also need metrics locally for a short duration for things like Kubernetes Horizontal Pod Scaling with custom metrics. We are running a prometheus setup with 25 cores and 220Gi of memory and our calculations suggest that the OTEL collector would only need half the resource usage across multiple replicas. Unfortunately, we would still likely need to maintain a prometheus cluster as well to maintain the metrics needed for scaling.

Did your calculations take into account the storage and querying of that data, or did they only consider data collection?

Even though this seems to be more of an Otel question than a Prometheus one, I’ll convert this into a discussion.
